src/calculators/BinaryVotingPowerEarningPowerCalculator.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.23;
+
+import {BinaryEligibilityOracleEarningPowerCalculator} from
+  "./BinaryEligibilityOracleEarningPowerCalculator.sol";
+import {IVotes} from "@openzeppelin/contracts/governance/utils/IVotes.sol";
+
+contract BinaryVotingPowerEarningPowerCalculator is BinaryEligibilityOracleEarningPowerCalculator {
+  uint48 public votingPowerUpdateFrequency;
+  uint48 public immutable UPDATE_START_TIME;
+  address public immutable VOTING_POWER_TOKEN;
+
+  constructor(
+    address _owner,
+    address _scoreOracle,
+    uint256 _staleOracleWindow,
+    address _oraclePauseGuardian,
+    uint256 _delegateeScoreEligibilityThreshold,
+    uint256 _updateEligibilityDelay,
+    uint48 _votingPowerUpdateFrequency,
+    address _votingPowerToken
+  )
+    BinaryEligibilityOracleEarningPowerCalculator(
+      _owner,
+      _scoreOracle,
+      _staleOracleWindow,
+      _oraclePauseGuardian,
+      _delegateeScoreEligibilityThreshold,
+      _updateEligibilityDelay
+    )
+  {
+    _setVotingPowerUpdateFrequency(_votingPowerUpdateFrequency);
+    UPDATE_START_TIME = uint48(block.number);
+    VOTING_POWER_TOKEN = _votingPowerToken;
+  }
+
+  function getEarningPower(uint256, /* _amountStaked */ address, /* _staker */ address _delegatee)
+    external
+    view
+    virtual
+    override
+    returns (uint256)
+  {
+    uint48 _votingPowerTimepoint = _getVotingPowerTimepoint();
+    uint256 _votingPower =
+      IVotes(VOTING_POWER_TOKEN).getPastVotes(_delegatee, _votingPowerTimepoint);
+
+    if (_isOracleStale() || isOraclePaused) return _votingPower;
```

Comment:
Keep this but read other epc, from staker

Reply:
I think the address we'd need to look at, the address which represents the delegate in this case, is the one passed as the `_staker` to the EPC, i.e. the owner of the "deposit" in the DelegateCompensationStaker, right?

Reply:
This is a good question. I updated the owner and delegate to be the same address for both spikes so it shouldn't matter, but it may matter for future implementations. Which would be better? Happy to choose either.

Reply:
Let's make the EPC methods public on BinaryEligibilityEPC, add to existing pr, add tests for new public methods